### PR TITLE
just added lab3.c to git

### PR DESCRIPTION
**Pull request by IgorBrat**

### CODE REVIEW

File: lab3.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <string.h>
+
+void reverseString(char s[])
```
Comment (RoksolanaMazurak) [inline lab3.c]: please add comment to the function

**Review by RoksolanaMazurak — CHANGES_REQUESTED** [review]

File: lab3.c (hunk 1 above)
Comment (IgorBrat) [reply lab3.c]: fixed

File: lab3.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <string.h>
+
+void reverseString(char s[])     // function which reverses inputed string
```
Comment (zveres-ss) [inline lab3.c]:
```suggestion
void reverseString(char input_string[])     // function which reverses inputed string
```

File: lab3.c [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		temp = s[i];
+		s[i] = s[j];
+		s[j] = temp;
+	}
+}
+
+void main(void)
+{
+	printf("Input string: ");
+	char str[100];
```
Comment (zveres-ss) [inline lab3.c]: make 100 as constant

**Review by zveres-ss — CHANGES_REQUESTED** [review]

File: lab3.c (hunk 2 above)
Comment (IgorBrat) [reply lab3.c]: Done

File: lab3.c (hunk 3 above)
Comment (IgorBrat) [reply lab3.c]: Done

File: lab3.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```
Comment (peopleAlreadyKnowWhoIAm) [inline lab3.c:1]: Make this program hack pentagon

**Review by peopleAlreadyKnowWhoIAm — CHANGES_REQUESTED** [review]

**Pull request merged**